### ISSUE TITLE
Could I ask you please provide documantaion and/or example of usage?

### ISSUE THREAD

**Issue by vit1251** [neurlang/wayland#8]
Hi dude, look like you create an well and wide solution but for start it require examples with create widnow, button and some additonal extra tutorials. Right now look like very raw solution and I does not have extra time to learn infrastructure and reference between all of them. COuld you lease provide examples for usage that bindning?

**Comment by neurlang** [comment]
You are right that this is a low level library for opening windows and drawing to texture. Any more user interface elements (buttons etc) won't be provided, the user is expected to draw them on their own or use some other library.
